cfme/common/__init__.py

```diff
@@ -85,6 +86,75 @@ def _assign_unassign_policy_profiles(self, assign, *policy_profile_names):
         details_view = self.create_view(navigator.get_class(self, 'Details').VIEW)
         details_view.flash.assert_no_error()
 
+    def assign_policy_profiles_multiple_entities(self, entities, *policy_profile_names):
+        """ Assign Policy Profiles to selected entity's on Collection All view
+
+        Args:
+            entities: list of entity's from collection table
+            policy_profile_names: :py:class:`str` with Policy Profile names. After Control/Explorer
+                coverage goes in, PolicyProfile objects will be also passable.
+        """
+        map(self.assigned_policy_profiles.add, policy_profile_names)
```

Comment:
**REQUIRED**: A few questions I would like @mfalesni and @jkrocil to weigh in on too:
1) This shouldn't be a normal method. I would think a class method would be better.
2) Personally I think we should be passing objects here, both for the entities and the policy profiles. I know we don't necessarily do this now (on this PR), but I'm bringing it up because I think we should consider it.

Reply:
Hi @psav ; @mfalesni and @jkrocil 
This method called by Container Image collection instance, from my test.
It won't work if i will change it to class method.
Following our conversation from last week, please consider to merge this PR as is.

Reply:
@psav Regarding step 1 that related to my PR,
Following our conversation from last week , please consider to accept this method as normal method.
I call assign_policy_profiles_multiple_entities from my test with instance of Image collection.
If i change this method to class method i will have to also pass the instance of Image collection as attribute.
Also all the other old methods like my in this class are normal methods.

Reply:
We have decided to implement as is for now